matchers.json

```diff
@@ -4,11 +4,11 @@
             "owner": "go",
             "pattern": [
                 {
-                    "regexp": "^([^:]*: )?((.:)?[^:]*):(\\d+)(:(\\d+))?: (.*)$",
-                    "file": 2,
-                    "line": 4,
-                    "column": 6,
-                    "message": 7
+                    "regexp": "((?:\\.{0,2}\\/|\\.{0,2}\\\\)(?:.+\\.go)):(?:(\\d+):(\\d+):)? (.*)",
```

Comment:
nit: It looks like you might be able to eliminate some groups. I think this is equivalent:

```
"(\\.{0,2}[\\/\\\\].+\\.go):(?:(\\d+):(\\d+):)? (.*)",
```

- moved `\\.{0,2}` in front of the group
- then switched `(?:\\/|\\\\)` to a char set `[\\/\\\\]` - arguably simpler? idk
- Removed group around the rest of the file path `(?:.+\\.go)` =\> `.+\\.go`

Reply:
Yeah, I think that is better.  I'll change

Reply:
I changed it to this.